integration/schema-expected.graphql

```diff
@@ -32,28 +29,29 @@ enum ErrorType {
 
 input ListCoercion {
   enumVal: [ErrorType]
-  strVal: [String]
   intVal: [Int]
   scalarVal: [Map]
+  strVal: [String]
 }
 
+""""""
```

Comment:
Why's this happened?

Reply:
check https://github.com/99designs/gqlgen/pull/2008